airflow/utils/types.py

```diff
@@ -42,8 +40,13 @@ class DagRunType(str, enum.Enum):
     def __str__(self) -> str:
         return self.value
 
-    def generate_run_id(self, logical_date: datetime) -> str:
-        return f"{self}__{logical_date.isoformat()}"
+    def generate_run_id(self, *, run_after: str) -> str:
+        """
+        Generate a random string for DagRun.
```

Comment:
We might need to update the docstring here

Reply:
changed it